frontend/src/app/users/users-profile/users-profile.component.html

```diff
@@ -0,0 +1,27 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<h2>Profile of {{ userName }}</h2>
```

Comment:
This page looks very sad: 

![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/c61cd803-fc89-42c3-be10-ab21432d7b2b)

Can you optimize the UI a bit? A few ideas: 
- Add a profile image (just an avatar placeholder)
- Indicate if it's the own profile
- Add the year when the user joined

Reply:
![Bildschirmfoto 2023-11-14 um 09 42 35](https://github.com/DSD-DBS/capella-collab-manager/assets/49535291/ffe8e41d-b404-4c05-bf68-807c8847043e)
looks like that currently

Reply:
I'd move the avatar to the left side. 
Also, we should only indicate the year when the user joined, not the full date (and this date is optional, you should consider it, some users don't have the date).